### PR TITLE
processor(ticdc): fix some bugs about pull-based sink

### PR DESCRIPTION
**Pull request by hicqu**
Signed-off-by: qupeng <qupeng@pingcap.com>

### What problem does this PR solve?

Issue Number: ref #5928 

### What is changed and how it works?

Fix some bugs about pull-based sink, introduced in some latest PRs.

#### Tests 

 - Unit test

### Release note 

```release-note
None
```

### CODE REVIEW

File: cdc/processor/sinkmanager/worker_impl.go [hunk 1]
```diff
@@ -309,12 +317,20 @@ func (w *sinkWorkerImpl) appendEventsToTableSink(t *sinkTask, events []*model.Po
 }
 
 func (w *sinkWorkerImpl) advanceTableSink(t *sinkTask, commitTs model.Ts, size uint64, batchID uint64) error {
+	log.Info("Advance table sink",
```
Comment (nongfushanquan) [inline cdc/processor/sinkmanager/worker_impl.go:320]: log.Debug?

**Review by nongfushanquan — APPROVED** [review]

File: cdc/processor/processor.go [hunk 2]
```diff
@@ -865,6 +865,7 @@ func (p *processor) lazyInitImpl(ctx cdcContext.Context) error {
 		p.sourceManger = sourcemanager.New(p.changefeedID, p.upstream, sortEngine, p.errCh)
 		sinkManager, err := sinkmanager.New(stdCtx, p.changefeedID, p.changefeed.Info, p.redoManager,
 			sortEngine, p.mg, p.errCh, p.metricsTableSinkTotalRows)
+		p.sourceManger.OnResolve(sinkManager.UpdateReceivedSorterResolvedTs)
```
Comment (hi-rustin) [inline cdc/processor/processor.go:868]: Could you please add a comment to highlight this binding?

File: cdc/processor/sinkmanager/worker_impl.go (hunk 1 above)
Comment (hi-rustin) [reply cdc/processor/sinkmanager/worker_impl.go]: +1

**Review by hi-rustin — APPROVED** [review]
Thanks!

**Pull request merged**